src/policy_evaluator.rs

```diff
@@ -181,7 +124,11 @@ impl PolicyEvaluator {
                     Policy::new,
                     policy_execution_mode,
                 )?;
-                let evaluator = burrego::opa::wasm::Evaluator::new(id, &policy_contents)?;
+                let evaluator = burrego::opa::wasm::Evaluator::new(
+                    id,
+                    &policy_contents,
+                    &opa_callbacks::DEFAULT_HOST_CALLBACKS,
```

Comment:
Here we should not use the default host callbacks of burrego, but rather provide ours (which would using the `tracing` crate among other things).